gu-base/src/daemon_lib.rs

```diff
@@ -18,80 +18,73 @@ impl GuServer {
     }
 }
 
-#[derive(PartialEq)]
-enum DaemonCommand {
+#[derive(PartialEq, Clone, Copy, Debug)]
+pub enum DaemonCommand {
     None,
     Run,
     Start,
     Stop,
     Status,
 }
 
-pub struct DaemonModule {
+pub struct DaemonHandler {
```

Comment:
CLI subcommand is `"server"`, so I'd rename it to, as 
```suggestion
pub struct ServerHandler {
```